.github/workflows/ci.yml

```
@@ -184,24 +184,24 @@ jobs:
             end
 
             File.open(ENV.fetch("GITHUB_ENV"), "a") do |f|
-              f.puts "CASK_CONFLICTS=\"#{cask_conflicts&.join(" ")}\"" if cask_conflicts.present?
-              f.puts "CASK_DEPENDENCIES=\"#{cask_dependencies&.join(" ")}\"" if cask_dependencies.present?
+              f.puts "CASK_CONFLICTS=#{cask_conflicts&.join(' ')}" if cask_conflicts.present?
+              f.puts "CASK_DEPENDENCIES=#{cask_dependencies&.join(' ')}" if cask_dependencies.present?
```

```suggestion
              f.puts "CASK_CONFLICTS=#{cask_conflicts&.join(" ")}" if cask_conflicts.present?
              f.puts "CASK_DEPENDENCIES=#{cask_dependencies&.join(" ")}" if cask_dependencies.present?
```

These can continue to use double quotes for the `#join` argument, since this code is being interpolated.